Dockerfile

```diff
@@ -1,24 +1,25 @@
-FROM node:16-alpine AS builder
+FROM node:20.5.0-alpine AS deps
```

Comment:
This could be read from nvm but then we are creating our own node Docker image that I did not see necessary. I think that this could be avoided if changing from dependabot to renovate with automatic dependency updates.